doc/reference/tooling/tt_cli/cluster.rst

```diff
@@ -271,6 +272,74 @@ to ``ro`` and reloads the configuration.
     replica set members, so there can be any number of read-write instances in one replica set.
 
 
+.. _tt-cluster-failover:
+
+failover
+--------
+
+.. code-block:: console
+
+    $ tt cluster failover SUBCOMMAND [OPTION ...]
+
+``tt cluster failover`` manages a :ref:`supervised failover <repl_supervised_failover>` in Tarantool clusters.
+
+-   :ref:`switch <tt-cluster-failover-switch>`
+-   :ref:`switch-status <tt-cluster-failover-switch-status>`
+
+.. important::
+
+    ``tt cluster failover`` works only with centralized cluster configurations stored in etcd.
+
+
+.. _tt-cluster-failover-switch:
+
+switch
+~~~~~~
+
+.. code-block:: console
+
+    $ tt cluster failover switch URI INSTANCE_NAME [OPTION ...]
+
+``tt cluster failover switch`` appoints the specified instance to be a master.
+This command accepts the following arguments and options:
+
+-   ``URI``: A :ref:`URI <tt-cluster-uri>` of the cluster configuration storage.
+-   ``INSTANCE_NAME``: An instance name.
+-   ``[OPTION ...]``: :ref:`Options <tt-cluster-options>` to pass to the command.
+
+Example:
+
+.. code-block:: console
+
+    $ tt cluster failover switch http://localhost:2379/myapp storage-a-002
+    To check the switching status, run:
+    tt cluster failover switch-status http://localhost:2379/myapp b1e938dd-2867-46ab-acc4-3232c2ef7ffe
+
+
+.. _tt-cluster-failover-switch-status:
+
+switch-status
+~~~~~~~~~~~~~
+
+.. code-block:: console
+
+    $ tt cluster failover switch-status URI TASK_ID
+
+``tt cluster failover switch-status`` shows the status of switching a master instance.
```

Comment:
What statuses do we have?

Reply:
Not sure that we need to list possible statuses. Looks like the command output is useful and doesn't require additional explanations:

![image](https://github.com/tarantool/doc/assets/38073144/1ceb4d45-a924-4c0a-9c04-a10dc3172e9f)